### PR TITLE
zeroIfLessThanZero

### PR DESCRIPTION
**Pull request by farahwalid**

### CODE REVIEW

File: SWEgit/src/com/company/Main.java [hunk 1]
```diff
@@ -47,9 +47,28 @@ public static void Get_Median(String []arr) {
...
+    public static void zeroIfLessThanZero(String[] arr) throws NumberFormatException{
+        int temp[]=new int[arr.length];
+        try {
+            for (int i = 0; i < arr.length; i++) {
+                temp[i] = Integer.parseInt(arr[i]);
+            }
+        }
+        catch (NumberFormatException e){
+            e.printStackTrace();
+        }
```
Comment (EhabMostafa) [inline SWEgit/src/com/company/Main.java:61]: I suggest to print error message not e.printStachTrace ();

Comment (farahwalid) [reply SWEgit/src/com/company/Main.java:61]: da ghalt y3ni ?

Comment (EhabMostafa) [reply SWEgit/src/com/company/Main.java:61]: I suggest
e.printStackTrace();   ->>> System.out.println("zeroIfLessThanZero : Sorry this function run with array of numbers only !!");

Comment (EhabMostafa) [reply SWEgit/src/com/company/Main.java:61]: and after this line in catch add this return;

**Pull request closed without merging**